demos/motion_tracking/Readme.md

```diff
@@ -0,0 +1,34 @@
+# Dynamic Neural Fields - Demos
+
+This readme assumes that you have installed lava, lava-loihi and lava-dnf in the same virtual environment.
+
+### Running the demos
+The demo will run in your browser via port-forwarding. Choose a random port_num between 10000 and 20000.
+(This is to avoid that multiple users try to use the same port)
+
+#### Connect to external vlab with port-forwarding
+```bash
+ssh <my-vm>.research.intel-research.net -L 127.0.0.1:<port_num>:127.0.0.1:<port_num>
+```
+
+#### Activate your virtual environment
+- depends on where your venv is located, e.g.:
+```bash
+source .venv/bin/activate
+```
+
+#### Navigate to the motion_tracking demo:
+```bash
+cd lava-dnf/demos/motion_tracking
+```
+#### Start the bokeh app
+```bash
+SLURM=1 LOIHI_GEN=N3C1 PARTITION=kp bokeh serve app.py --port <port_num>
```

Comment:
I wonder whether this is still necessary with the new API.
Could you check whether you can do
```bash
bokeh serve main_motion_tracking.py --port <port_num>
```
when you add
```python
from lava.utils import loihi
loihi.use_slurm_host()
```
to the beginning of `main_motion_tracking.py`?
You may have to specify `partition`, `board`, or `loihi_gen` in the call to `use_slurm_host()` but I don't think that is necessary in this case.

Reply:
Done